package-lock.json

```diff
@@ -1,627 +1,8 @@
 {
   "name": "nightingale",
-  "version": "2.3.1",
-  "lockfileVersion": 2,
+  "version": "2.3.3",
+  "lockfileVersion": 1,
   "requires": true,
-  "packages": {
```

Comment:
There has been quite a lot of package lock content removed, why?

Reply:
This wasnt intentional as i did not manually change that file. I dont know why it happened. Do you have any recommendations on how to solve this?